### PR TITLE
just testing that the boolean is preserved on gha

### PR DESCRIPTION
**Pull request by emma-nechamkin**

### CODE REVIEW

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Comment by emma-nechamkin** [comment]
fyi @esfoobar-usds this is not going to get merged -- i'm just testing to see why what i have when i run this locally is so different from what is in the file vim downloads

**Comment by esfoobar-usds** [comment]
Oh ok.

File: data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py [hunk 1]
```diff
@@ -375,7 +379,7 @@ def write_esri_shapefile():
                 for task in [
                     write_high_to_file,
                     write_low_to_file,
-                    # write_esri_shapefile,
+                    write_esri_shapefile,
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py:382]: we shouldn't have had this nulled?

**Pull request merged**